kubernetes-maven-plugin/doc/src/main/asciidoc/inc/image/_naming.adoc

```diff
@@ -1,5 +1,5 @@
 [[image-name]]
-== Image Names
+.Image Names
```

Comment:
Why is this no longer a heading? did the document structure change?

Reply:
move the label (and maybe the link reference) to the top of the table where it makes sense (As a Figure title), following the _configuration.adoc pattern